cmake/modules/RootBuildOptions.cmake

```diff
@@ -189,6 +189,9 @@ option(minimal "Enable only required options by default" OFF)
 option(rootbench "Build rootbench if rootbench exists in root or if it is a sibling directory." OFF)
 option(roottest "Build roottest if roottest exists in root or if it is a sibling directory." OFF)
 option(testing "Enable testing with CTest" OFF)
+option(asan "Build ROOT with address sanitizer instrumentation" OFF)
+include(CMakeDependentOption)
```

Comment:
Can we move include at the beginning of file and can you add comment why you included it (for the future)